contrib/docker/Dockerfile

```
@@ -26,12 +26,15 @@ RUN if [ -z "$ZCASH_VERSION" ]; \
 RUN useradd --home-dir /srv/$ZCASHD_USER \
```

```suggestion
RUN useradd --home-dir "/srv/$ZCASHD_USER" \
```
and similarly on lines 16 and 22.